pkg/code-generator/codegen/generator.go

```diff
@@ -141,7 +141,7 @@ func generateFilesForResourceGroup(rg *model.ResourceGroup) (code_generator.File
 	} {
 		content, err := generateResourceGroupFile(rg, tmpl)
 		if err != nil {
-			return nil, errors.Wrapf(err, "internal error: processing %template '%v' for resource group %v failed", tmpl.ParseName, rg.Name)
+			return nil, errors.Wrapf(err, "internal error: processing %template '%s' for resource group %v failed", tmpl.ParseName, rg.Name)
```

Comment:
what's the "%t"?

Reply:
Hahaha
That’s been there’s while I guess
Probably a mistake